pkg/result/processors/nolint.go

```diff
@@ -252,16 +252,19 @@ func (p *Nolint) extractInlineRangeFromComment(text string, g ast.Node, fset *to
 		}
 	}
 
-	if !strings.HasPrefix(text, "nolint:") {
+	if strings.HasPrefix(text, "nolint:all") || !strings.HasPrefix(text, "nolint:") {
 		return buildRange(nil) // ignore all linters
 	}
 
 	// ignore specific linters
 	var linters []string
 	text = strings.Split(text, "//")[0] // allow another comment after this comment
 	linterItems := strings.Split(strings.TrimPrefix(text, "nolint:"), ",")
-	for _, linter := range linterItems {
-		linterName := strings.ToLower(strings.TrimSpace(linter))
+	for _, item := range linterItems {
+		linterName := strings.ToLower(strings.TrimSpace(item))
+		if linterName == "all" {
+			panic("OOPS")
```

Comment:
Maybe a more explicit message would be better? Is OOPS an acronym, or is an exclamation made in response to a minor mistake?

Reply:
for example

> incorrect usage of "all" keyword, please use it only with "nolint:all"

Reply:
It's just a test residue